release-1.8/release_notes_draft.md

```diff
@@ -72,22 +72,43 @@ fundamental aspect of a secure cluster.
   * The`--audit-policy-file` requires `kind` and `apiVersion` fields
     specifying what format version the `Policy` is using.
 
+* The deprecated ThirdPartyResource (TPR) API has been removed.
+  To avoid losing your TPR data, you must
+  [migrate to CustomResourceDefinition](https://kubernetes.io/docs/tasks/access-kubernetes-api/migrate-third-party-resource/)
+  before upgrading.
+
+* The following deprecated flags have been removed from `kube-controller-manager`:
```

Comment:
The following deprecated flags have been removed from `kube-controller-manager`:

* `replication-controller-lookup-cache-size`
* `replicaset-lookup-cache-size`
* `daemonset-lookup-cache-size`